permission_handler_platform_interface/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 3.12.0
+
+* Adds `Permission.calendarReadOnly` and `PermissionCalendarFullAccess`.
+* Deprecates `Permission.calendar`. Developers should use `Permission.calendarReadOnly` and `PermissionCalendarFullAccess` instead.
```

Comment:
Why is there a difference in convention between `Permission.calendarReadOnly` and `PermissionCalendarFullAccess`?